.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,16 @@
 Please fill in this template.
 
-- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
-- [ ] Test the change in your own code. (Compile and run.)
-- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
-- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
-- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
-- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
+- [x] Use a meaningful title for the pull request. Include the name of the package modified.
```

Comment:
Please revert your edits to this file, thanks.

Reply:
Sorry, didn't realize until after creating the PR that this file was part of the template. It should be reverted now.